obs/project.go

```diff
@@ -91,3 +97,102 @@ type RepositoryPath struct {
 	Project    string `json:"project" xml:"project,attr"`
 	Repository string `json:"repository" xml:"repository,attr"`
 }
+
+func (c Client) CreateProject(project *Project) error {
+	xmlData, err := xml.MarshalIndent(project, "", " ")
+	if err != nil {
+		return err
+	}
+
+	urlPath, err := url.JoinPath(c.APIURL, "source", project.Name, "_meta")
+	if err != nil {
+		log.Fatal(err)
```

Comment:
We should return an error here, similar to what I described in the previous comment:

```suggestion
		return fmt.Errorf("creating obs project: joining url: %w", err)
```